Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,239 @@
 /*
-Project Name: HTML Template
-Client: Your Client
-Author: Your Name
-Developer: @poopsplat in Chicago
+Project Name: Landing Page
+Client: Relaxr
+Author: Haley 
+Developer: @needleinthehaley in Chicago
 */
 
 /******************************************
-/* BASE STYLES
+/* UTILITY
 /*******************************************/
-body {
-  color: black;
-  font-size: 16px;
-  background: white;
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
 }
 
+/*i started off with ALL intentions of utilizing the categories 
+outlined in the CSS template. that is SO hard to do when you are
+constantly trying to figure everything out. i didn't do it this 
+time, but if i have time i'll go back in and sort. i'm sure 
+duplication of code is annoying to review.*/
```

Comment:
Totally — it's very easy for CSS to get out of control since you're going back and forth with it so frequently, so it's just a matter of getting comfortable with it, and being vigilant about removing unneeded things, moving things to where they should go, and formatting **when** those things come up, because if you "save it until later", it probably won't happen, haha. :)